final_prep/README.md

```diff
@@ -19,37 +19,40 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
-
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- _I will update anything I need to when the 2107 Cohort calendar is available._
+- Week 1 Screenshot
+![](https://user-images.githubusercontent.com/77412134/115662265-5c612880-a304-11eb-960e-728945fee7fb.png)
+- Week 2 Screenshot
+![](https://user-images.githubusercontent.com/77412134/115662513-b4982a80-a304-11eb-9e40-fe6a1ae83f4d.png)
+- Week 3 Screenshot
+![](https://user-images.githubusercontent.com/77412134/115662574-ca0d5480-a304-11eb-9b9f-19abd0c9be33.png)
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: [My Gist](https://gist.github.com/TannerDale/05e089e3bf3e7016f758493d8e8c9c55)
```

Comment:
Nice job- I like how you've outlined your experience. You may also think about including something about what you hope to work on with a mentor, or how you imagine best working together.

Reply:
I will make sure to do that! Thanks